dspace-api/src/main/java/org/dspace/content/DCSeriesNumber.java

```diff
@@ -81,7 +81,7 @@ public String toString()
         {
             return (null);
         }
-        else if (number == null)
+        else if (number == null || number.replaceAll("\\s","").length() == 0)
```

Comment:
To check for a string containing only spaces, it's more logical to just use the `matches` method imo.

`   else if (number == null || number.matches("^\s*$")`